piton/tools/src/sims/metro_chipset.config

```diff
@@ -138,11 +138,30 @@
     -vcs_build_args=+nospecify
     //-vcs_build_args=+nbaopt
     -vcs_build_args=-Xstrict=1 -notice
-    -vcs_build_args=-P $DV_ROOT/tools/pli/iop/bwioj.tab // needed for fake_l2
-    -vcs_build_args=-P $DV_ROOT/tools/pli/socket/bwsocket_pli.tab
-    -vcs_build_args=-P $DV_ROOT/tools/pli/mem/bwmem_pli.tab
-    -vcs_build_args=-lsocket_pli -liob -lmem_pli
+    //-vcs_build_args=-P $DV_ROOT/tools/pli/iop/bwioj.tab // needed for fake_l2
+    //-vcs_build_args=-P $DV_ROOT/tools/pli/socket/bwsocket_pli.tab
+    //-vcs_build_args=-P $DV_ROOT/tools/pli/mem/bwmem_pli.tab
+    //-vcs_build_args=-lsocket_pli -liob -lmem_pli
```

Comment:
At this point we probably ought to add a preprocessor macro for DPI stuff... let's leave this for now though

Reply:
Refactor done